test/note_test.rb

```diff
@@ -5,16 +5,56 @@ def setup
     @frequency   = 220.to_f
     @duration    = 2.0
     @sample_rate = 22050
+    @total_frames = (@duration * @sample_rate).to_i
```

Comment:
you'll have fewer instance vars if you do the setup for individual tests in the method for that test instead of here.